release-kubeapps-package.sh

```diff
@@ -0,0 +1,104 @@
+#! /usr/bin/env bash
+
+# Copyright 2022 the Kubeapps contributors.
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Variables
+readonly script_name=$(basename "${0}")
+readonly script_dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+readonly template_dir="$script_dir/packaging-templates"
+readonly git_repo="vmware-tanzu/package-for-kubeapps"
+
+# Variables set by get_options.
+version=""
+packaging_version_suffix=""
+
+main() {
+  get_options "${@}"
+  
+  # Commit, tag and create a release for production only.
+  echo "Releasing Kubeapps Carvel package v$version to repo '$git_repo'"
+  create_release "$version" "$packaging_version_suffix"
+}
+
+# Commit, tag and create a release in GitHub.
+create_release() {
+  local version=$1
+  local packaging_version_suffix=$2
+  local version_with_suffix="$version$packaging_version_suffix"
+  local tag="v$version_with_suffix"
+  git add "./$version_with_suffix"
+  git commit -m "Adding $tag files"
```

Comment:
Should we also add the `-s` and `-S` flags to DCO and GPG-sign the commits and tags?

Reply:
Agree, added issue #31 to do it in a separate PR.